docs/src/main/asciidoc/_attributes.adoc

```diff
@@ -1,6 +1,8 @@
+// Common attributes. 
+// --> No blank lines (it ends the document header)
```

Comment:
so these were ignored until now?

Reply:
No. things were borked, and nothing cared.